### PR TITLE
Add Scala Symposium 2017 event

### PR DESCRIPTION
**Pull request by b-studios**
As suggested by @SethTisue via Twitter, this PR adds the Scala Symposium event.

@blaisorblade Maybe you want to double check, just to be sure?

### CODE REVIEW

**Comment by heathermiller** [comment]
(Also, thank you for making this suggestion @SethTisue. I should've thought to suggest this to them since I've been in email correspondence with the guys, doh!)

File: events/_posts/2017-10-23-scala-symposium.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+category: event
+title: Scala Symposium 2017
+logo: /resources/img/scala2017.png
+location: Vancouver, Canada
+description: "A scientific forum for researchers and practicioners focused on the Scala language."
```
Comment (Blaisorblade) [inline events/_posts/2017-10-23-scala-symposium.md]: `s/practicioners/practitioners/` (Sorry).

**Comment by Blaisorblade** [comment]
LGTM (apart for typo)! Just got the same suggestion after pinging Jon Pretty.

**Comment by SethTisue** [comment]
fast track merge since scala-lang just tweeted the URL. typo fix in 5834fda44bfc8dd0278cdaf4e96c0259882d47ae

**Pull request merged**